x/oracle/keeper/msg_server.go

```diff
@@ -28,7 +28,7 @@ func (k Keeper) DelegateFeedConsent(c context.Context, msg *types.MsgDelegateFee
 		return nil, sdkerrors.Wrap(stakingtypes.ErrNoValidatorFound, sdk.ValAddress(val).String())
 	}
 
-	if validator.IsUnbonded() {
+	if ctx.BlockHeight() > 0 && validator.IsUnbonded() {
```

Comment:
why is this?

Reply:
Because validators are unbonded when the gentx runs at block 0. Special casing for genesis bootstrapping fixes this.

Reply:
does this also apply to non-zero height upgrades? It'd be helpful to add that as a comment on the code